.github/scripts/poll_for_endpoint_health.js

```diff
@@ -0,0 +1,45 @@
+import fetch from 'node-fetch';
+import smokeConsoleTestnet from '../../config/console-smoke-testnet.js';
+import smokeConsoleMainnet from '../../config/console-smoke-mainnet.js';
+
+const networkArg = process.argv[2];
+
+if (!networkArg || (networkArg !== 'testnet' && networkArg !== 'mainnet')) {
+  console.error('Usage: node script.js [testnet/mainnet]');
+  process.exit(1);
+}
+
+const endpointUrl = networkArg === 'testnet' ? smokeConsoleTestnet.network.console : smokeConsoleMainnet.network.console;
```

Comment:
Possibly makes more sense, though is async so adds some complexity. Will let you make a call

```suggestion
const endpointUrl = await import(path + networkArg).default.network.console
```

Reply:
I think it is simple enough for now to leave, if we was supporting more networks I would change